libraries/Wire/src/Wire.cpp

```diff
@@ -403,44 +394,48 @@ void TwoWire::flush(void)
 }
 
 // behind the scenes function that is called when data is received
-void TwoWire::onReceiveService(uint8_t *inBytes, int numBytes)
+void TwoWire::onReceiveService(i2c_t *obj, uint8_t *inBytes, int numBytes)
```

Comment:
I think it will be more concise to use the obj to get the inBytes and numBytes as they are fields of obj:
```
inBytes = (uint8_t *) obj->i2cTxRxBuffer
numBytes = obj->slaveRxNbData
```

```suggestion
void TwoWire::onReceiveService(i2c_t *obj)
```

Reply:
Done